src/Core/CoreKernel.php

```diff
@@ -122,6 +131,17 @@ protected function bootDatabaseEnvVars()
         $databaseConfig = $this->getDatabaseConfig();
         $databaseConfig['database'] = $this->getDatabaseName();
         DB::setConfig($databaseConfig);
+
```

Comment:
I think the "default" mentioned above should actually be "primary" (I can't comment on that line directly 'cause it wasn't touched)

Reply:
Updated